lib/boot.js

```diff
@@ -56,10 +56,14 @@ const client = initClient({
         return {
           ...note,
           contentKey: content
+            .replace(/^\s*#+\s*/, '') // Remove leading whitespace and Markdown heading
             .replace(/\s+/g, ' ')
             .trim()
             .slice(0, 200)
-            .toLowerCase(),
+            .toLowerCase()
+            // Remove accents/diacritics (https://stackoverflow.com/questions/990904)
+            .normalize('NFD')
```

Comment:
**TIL**

Reply:
Ugh, I just realized that this won't work in IE11 (because of course it doesn't). I'll add some conditional logic to skip this part, since the shim is huge and I don't want to include it just for this.

Reply:
@roundhill I [fixed this part](https://github.com/Automattic/simplenote-electron/pull/1144/commits/7737bea24c57f9e7ef98758099fb1fa3ec9a13fa#diff-a597bcf2c27edf0291e524f8897ad4f1R94) so it won't choke in IE11.